Mending CBM-related descriptions and dialogs

linzhongfeiniao opened the pull request:
#### Summary
None
#### Purpose of change
Mending several CBM-related descriptions and dialogs to make lines user-friendly.
#### Describe the solution

THIS PR will be in progress in a while and welcome everyone to co-operate and find not-clear lines to fix.
 
1.Use more standard/technical words to describe. This will maintain the style of the whole CBM lines. Example: Pack CBM in pouch --> Package CBM for implantation

2.More suitable words according to background settings. Example1: hominid palaeoanthropology  -->  human factors engineering biomechanics       NOTE: The previous one is obscure and misleading to some historical discoveries, not techs.  
Example2: Beqball --> dodgeball   NOTE: The ball game has to be related with THROWING Skills, so I tried to use the dodgeball in the real life.

3. For terms like 'Chiropteran Signal CBM' and 'Desmodontinae Gargantua Vocalizer' that has been in the game for a while, I want to salute and give respect to the original author's great imagination and want to maintain the original style. So I picked up the description of the two terms to give player clearer guidance on what the CBM is used for.

4.Fixing typos like inconsistent using of colons, grammas and so on.

5.Welcome to critics, especially for the changing of descriptions, for which I want to change more and really need advice.
 
#### Describe alternatives you've considered

<!-- Explain any alternative solutions, different approaches, or possibilities you've considered using to solve the same problem. -->

#### Testing

Only text changing. Wait for clang check of the review system.

#### Additional context

Keep working on. 
And I will focus on CBM things for a while.

<!--README: Cataclysm: Dark Days Ahead is released under the Creative Commons Attribution ShareAlike 3.0 license.
The code and content of the game are free to use, modify, and redistribute for any purpose whatsoever.
By contributing to the project you agree to the terms of the license and that any contribution you make will also be covered by the same license.
See http://creativecommons.org/licenses/by-sa/3.0/ for details. -->

linzhongfeiniao committed:
Mending descriptions for CBM items

Updated descriptions for Chiropteran Signal and Desmodontinae Gargantua Vocalizer CBMs to provide more detail about their functions and enhancements.

linzhongfeiniao committed:
Fix typo in CBM recharge and reserve rules labels

linzhongfeiniao committed:
Rename install options in the menu

linzhongfeiniao committed:
Update name for PACK_CBM item action

linzhongfeiniao committed:
Revise description for Throwing Assist CBM

Updated the description of the Throwing Assist CBM to reflect changes in terminology and context.

linzhongfeiniao committed:
Modify description terminology in electronics.json

Updated the description terminology from 'Anglic' to 'Interstellar Standard'.

linzhongfeiniao committed:
Merge branch 'CleverRaven:master' into linzhongfeiniao-patch-2-1

linzhongfeiniao committed:
Add 'doge' to the spell checker dictionary

linzhongfeiniao committed:
Revise CBM descriptions for clarity and detail

Updated descriptions for Chiropteran Signal CBM and Desmodontinae Gargantua Vocalizer CBM to replace 'sound' with 'vibrations' and clarify the nature of the clicking signal.

linzhongfeiniao committed:
Reorganize bionic_data structure and improve comments

Refactor bionic_data structure by reorganizing fields and adding new comments for clarity.

TheShadowFerret commented on data/mods/Xedra_Evolved/items/cbms.json:
```diff
@@ -25,7 +25,7 @@
     "subtypes": [ "BIONIC_ITEM" ],
     "name": { "str": "Chiropteran Signal CBM" },
-    "description": "A jar-like container that includes a set of vocal cords with a large hood at the top of them and rootlike tendrils at the base connecting to a microchip.",
+    "description": "A jar-like CBM housing a modified set of vocal cords, capped by a broad resonating hood and threaded with rootlike tendrils that interface with an embedded microchip.  Once implanted, the system integrates with your own vocal apparatus, enhancing it to produce tightly controlled, bat-like resonance patterns. When activated, you are capable of producing unfamiliar vibrations.   ...
```
```suggestion
    "description": "A jar-like CBM housing a modified set of vocal cords, capped by a broad resonating hood and threaded with rootlike tendrils that interface with an embedded microchip.  Once implanted, the system integrates with your own vocal apparatus, enhancing it to produce tightly controlled, bat-like resonance patterns.  When activated, you are capable of producing unfamiliar vibrations.  It releases an ultrasonic clicking signal, which is mostly inaudible to humans, allowing you to commune with nearby bats and draw them into a defensive swarm.",
```

TheShadowFerret commented on data/mods/Xedra_Evolved/items/cbms.json:
```diff
@@ -34,8 +34,8 @@
-    "name": { "str": "Desmodontinae Gargantua Vocalizer" },
-    "description": "A jar-like container that includes a set of oversized and reinforced vocal cords with a large hood at the top of them and rootlike tendrils at the base connecting to a microchip.",
+    "name": { "str": "Desmodontinae Gargantua Vocalizer CBM" },
+    "description": "A jar-like CBM housing a modified set of oversized and reinforced vocal cords, capped by a broad and chitinous hood and threaded with rootlike tendrils that interface with an embedded microchip.  Once implanted, the system integrates with your own vocal apparatus, enhancing it to produce tightly controlled, bat-like resonance patterns. When activated, you are capable of produc ...
```
```suggestion
    "description": "A jar-like CBM housing a modified set of oversized and reinforced vocal cords, capped by a broad and chitinous hood and threaded with rootlike tendrils that interface with an embedded microchip.  Once implanted, the system integrates with your own vocal apparatus, enhancing it to produce tightly controlled, bat-like resonance patterns.  When activated, you are capable of producing unfamiliar vibrations.  It releases an ultrasonic clicking signal, which is mostly inaudible to humans, allowing you to commune with nearby giant bats and draw them into a defensive swarm.",
```

linzhongfeiniao commented:
OK,  I will put 'NO AI' in my rules. Sorry for all the inconveniences.

linzhongfeiniao committed:
Update description for Throwing Assist CBM

linzhongfeiniao committed:
Update data/mods/Xedra_Evolved/items/cbms.json

Co-authored-by: TheShadowFerret <99621099+TheShadowFerret@users.noreply.github.com>

linzhongfeiniao committed:
Remove 'doge' from the dictionary

linzhongfeiniao commented:
Rolled back on the ball game's name. Also removed what I shouldn't put in the dictionary. Updated the summary. 
I am working on my opening debate for my degree thesis, so recently I can't update the PR strictly on-time.
But I subscribed the email notification, so I can receive comments and critics. 
I need experiences like this to merge myself into the community. So, thanks anyway.

linzhongfeiniao committed:
Update description formatting in cbms.json

Fixed formatting of the description field for clarity.

linzhongfeiniao committed:
Reorder text typos

linzhongfeiniao committed:
Fix typo in disconnect message for player

linzhongfeiniao committed:
Fix grammatical errors in microscope snippet

linzhongfeiniao committed:
Fix description formatting for dinosaur_CBM.json

linzhongfeiniao committed:
Update comments for clarity in harvest_cbm.json

linzhongfeiniao committed:
Fix typos in zed-dinosaur_CBM.json

linzhongfeiniao committed:
Refine descriptions in renfield_cbms.json

Updated descriptions for various bionic modules to improve clarity and fix grammatical issues.

linzhongfeiniao committed:
Fix typo in bionics.json description

Corrected a typo in the description of the Expanded Digestive System bionic.

RenechCDDA commented:
All of the changes to bionics.h are totally unwarranted. That is a massive amount of churn, for no apparent reason.

linzhongfeiniao committed:
Roll back on bionics.h

linzhongfeiniao commented:
Rolled back on bionics.h.
Made too much unnecessary changes last time.
Have to figure out what is the most obvious one to change.

linzhongfeiniao committed:
Fix typo in bionic description for digestion

linzhongfeiniao committed:
Roll back on a 'who' use
